2025-04-02 08:01:42.675 +05:30 [INF] Application starting up
2025-04-02 08:01:45.863 +05:30 [INF] Main thread ID: 1
2025-04-02 08:01:45.863 +05:30 [INF] ARGS:- C:\BackOffice\ETL_Replacement\PIL\INTEL\Input\Source\Temp_Process\INTEL_01_v1_20231208_TELINV04_WIP.csv
2025-04-02 08:01:45.866 +05:30 [INF] Processing files for domain: PIL, invoiceType: INTEL
2025-04-02 08:01:49.620 +05:30 [WRN] Exception In TempInsertStoreProc: 42601: INSERT has more target columns than expressions
2025-04-02 08:01:49.697 +05:30 [ERR] Connection failed. Retrying in 5 seconds... Attempt 1
2025-04-02 08:01:49.697 +05:30 [ERR] Exception: 42601: INSERT has more target columns than expressions
2025-04-02 08:02:14.953 +05:30 [INF] Application starting up
2025-04-02 08:02:16.230 +05:30 [INF] Main thread ID: 1
2025-04-02 08:02:16.242 +05:30 [INF] ARGS:- C:\BackOffice\ETL_Replacement\PIL\INTEL\Input\Source\Temp_Process\INTEL_01_v1_20231208_TELINV04_WIP.csv
2025-04-02 08:02:16.247 +05:30 [INF] Processing files for domain: PIL, invoiceType: INTEL
2025-04-02 08:02:21.243 +05:30 [INF] Process file path :- C:\BackOffice\ETL_Replacement\PIL\INTEL\Input\Source\Temp_Process\INTEL_01_v1_20231208_TELINV04_WIP.csv
2025-04-02 08:02:21.243 +05:30 [INF] Process file name :- INTEL_01_v1_20231208_TELINV04_WIP.csv
2025-04-02 08:02:21.244 +05:30 [INF] Total time taken in process: 2.8952157 seconds
2025-04-02 08:02:21.247 +05:30 [INF] Total time taken in moving: 0.0027421 seconds
2025-04-02 08:16:56.849 +05:30 [INF] Application starting up
2025-04-02 08:16:57.813 +05:30 [INF] Main thread ID: 1
2025-04-02 08:16:57.813 +05:30 [INF] ARGS:- C:\BackOffice\ETL_Replacement\PIL\INTEL\Input\Source\Temp_Process\INTEL_01_v1_20231208_TELINV04_WIP.csv
2025-04-02 08:16:57.815 +05:30 [INF] Processing files for domain: PIL, invoiceType: INTEL
2025-04-02 08:17:00.688 +05:30 [WRN] Exception In TempInsertStoreProc: P0003: query returned more than one row
2025-04-02 08:17:00.761 +05:30 [ERR] Connection failed. Retrying in 5 seconds... Attempt 1
2025-04-02 08:17:00.762 +05:30 [ERR] Exception: P0003: query returned more than one row
2025-04-02 08:17:05.936 +05:30 [WRN] Exception In TempInsertStoreProc: P0003: query returned more than one row
2025-04-02 08:17:05.983 +05:30 [ERR] Connection failed. Retrying in 5 seconds... Attempt 2
2025-04-02 08:17:05.983 +05:30 [ERR] Exception: P0003: query returned more than one row
2025-04-02 08:17:15.177 +05:30 [INF] Application starting up
2025-04-02 08:17:16.029 +05:30 [INF] Main thread ID: 1
2025-04-02 08:17:16.029 +05:30 [INF] ARGS:- C:\BackOffice\ETL_Replacement\PIL\INTEL\Input\Source\Temp_Process\INTEL_01_v1_20231208_TELINV04_WIP.csv
2025-04-02 08:17:16.032 +05:30 [INF] Processing files for domain: PIL, invoiceType: INTEL
2025-04-02 08:17:18.852 +05:30 [WRN] Exception In TempInsertStoreProc: P0003: query returned more than one row
2025-04-02 08:17:18.928 +05:30 [ERR] Connection failed. Retrying in 5 seconds... Attempt 1
2025-04-02 08:17:18.928 +05:30 [ERR] Exception: P0003: query returned more than one row
2025-04-02 08:17:24.095 +05:30 [WRN] Exception In TempInsertStoreProc: P0003: query returned more than one row
2025-04-02 08:17:24.142 +05:30 [ERR] Connection failed. Retrying in 5 seconds... Attempt 2
2025-04-02 08:17:24.142 +05:30 [ERR] Exception: P0003: query returned more than one row
2025-04-02 08:17:29.272 +05:30 [WRN] Exception In TempInsertStoreProc: P0003: query returned more than one row
2025-04-02 08:17:29.317 +05:30 [ERR] Connection failed. Retrying in 5 seconds... Attempt 3
2025-04-02 08:17:29.317 +05:30 [ERR] Exception: P0003: query returned more than one row
2025-04-02 08:20:22.867 +05:30 [INF] Application starting up
2025-04-02 08:20:23.683 +05:30 [INF] Main thread ID: 1
2025-04-02 08:20:23.683 +05:30 [INF] ARGS:- C:\BackOffice\ETL_Replacement\PIL\INTEL\Input\Source\Temp_Process\INTEL_01_v1_20231208_TELINV04_WIP.csv
2025-04-02 08:20:23.686 +05:30 [INF] Processing files for domain: PIL, invoiceType: INTEL
2025-04-02 08:20:26.720 +05:30 [WRN] Exception In TempInsertStoreProc: 42883: operator does not exist: json >> unknown
2025-04-02 08:20:26.795 +05:30 [ERR] Connection failed. Retrying in 5 seconds... Attempt 1
2025-04-02 08:20:26.795 +05:30 [ERR] Exception: 42883: operator does not exist: json >> unknown
2025-04-02 08:20:31.991 +05:30 [WRN] Exception In TempInsertStoreProc: 42883: operator does not exist: json >> unknown
2025-04-02 08:20:32.039 +05:30 [ERR] Connection failed. Retrying in 5 seconds... Attempt 2
2025-04-02 08:20:32.039 +05:30 [ERR] Exception: 42883: operator does not exist: json >> unknown
2025-04-02 08:21:13.868 +05:30 [INF] Application starting up
2025-04-02 08:21:14.777 +05:30 [INF] Main thread ID: 1
2025-04-02 08:21:14.778 +05:30 [INF] ARGS:- C:\BackOffice\ETL_Replacement\PIL\INTEL\Input\Source\Temp_Process\INTEL_01_v1_20231208_TELINV04_WIP.csv
2025-04-02 08:21:14.781 +05:30 [INF] Processing files for domain: PIL, invoiceType: INTEL
2025-04-02 08:21:17.724 +05:30 [WRN] Exception In TempInsertStoreProc: 42883: operator does not exist: json >> unknown
2025-04-02 08:21:17.798 +05:30 [ERR] Connection failed. Retrying in 5 seconds... Attempt 1
2025-04-02 08:21:17.799 +05:30 [ERR] Exception: 42883: operator does not exist: json >> unknown
2025-04-02 08:21:22.998 +05:30 [WRN] Exception In TempInsertStoreProc: 42883: operator does not exist: json >> unknown
2025-04-02 08:21:23.043 +05:30 [ERR] Connection failed. Retrying in 5 seconds... Attempt 2
2025-04-02 08:21:23.043 +05:30 [ERR] Exception: 42883: operator does not exist: json >> unknown
2025-04-02 08:21:28.175 +05:30 [WRN] Exception In TempInsertStoreProc: 42883: operator does not exist: json >> unknown
2025-04-02 08:21:28.221 +05:30 [ERR] Connection failed. Retrying in 5 seconds... Attempt 3
2025-04-02 08:21:28.221 +05:30 [ERR] Exception: 42883: operator does not exist: json >> unknown
2025-04-02 08:21:33.098 +05:30 [INF] Application starting up
2025-04-02 08:21:34.154 +05:30 [INF] Main thread ID: 1
2025-04-02 08:21:34.155 +05:30 [INF] ARGS:- C:\BackOffice\ETL_Replacement\PIL\INTEL\Input\Source\Temp_Process\INTEL_01_v1_20231208_TELINV04_WIP.csv
2025-04-02 08:21:34.159 +05:30 [INF] Processing files for domain: PIL, invoiceType: INTEL
2025-04-02 08:21:37.008 +05:30 [WRN] Exception In TempInsertStoreProc: 42883: operator does not exist: json >> unknown
2025-04-02 08:21:37.079 +05:30 [ERR] Connection failed. Retrying in 5 seconds... Attempt 1
2025-04-02 08:21:37.079 +05:30 [ERR] Exception: 42883: operator does not exist: json >> unknown
2025-04-02 08:21:42.274 +05:30 [WRN] Exception In TempInsertStoreProc: 42883: operator does not exist: json >> unknown
2025-04-02 08:21:42.321 +05:30 [ERR] Connection failed. Retrying in 5 seconds... Attempt 2
2025-04-02 08:21:42.322 +05:30 [ERR] Exception: 42883: operator does not exist: json >> unknown
2025-04-02 08:21:47.496 +05:30 [WRN] Exception In TempInsertStoreProc: 42883: operator does not exist: json >> unknown
2025-04-02 08:21:47.546 +05:30 [ERR] Connection failed. Retrying in 5 seconds... Attempt 3
2025-04-02 08:21:47.546 +05:30 [ERR] Exception: 42883: operator does not exist: json >> unknown
2025-04-02 08:21:52.704 +05:30 [WRN] Exception In TempInsertStoreProc: 42883: operator does not exist: json >> unknown
2025-04-02 08:21:52.752 +05:30 [ERR] Connection failed. Retrying in 5 seconds... Attempt 4
2025-04-02 08:21:52.753 +05:30 [ERR] Exception: 42883: operator does not exist: json >> unknown
2025-04-02 08:21:57.937 +05:30 [WRN] Exception In TempInsertStoreProc: 42883: operator does not exist: json >> unknown
2025-04-02 08:21:57.986 +05:30 [ERR] Connection failed. Retrying in 5 seconds... Attempt 5
2025-04-02 08:21:57.987 +05:30 [ERR] Exception: 42883: operator does not exist: json >> unknown
2025-04-02 08:22:03.157 +05:30 [WRN] Exception In TempInsertStoreProc: 42883: operator does not exist: json >> unknown
2025-04-02 08:22:03.208 +05:30 [ERR] Connection failed. Retrying in 5 seconds... Attempt 6
2025-04-02 08:22:03.208 +05:30 [ERR] Exception: 42883: operator does not exist: json >> unknown
2025-04-02 08:22:08.424 +05:30 [INF] Process file path :- C:\BackOffice\ETL_Replacement\PIL\INTEL\Input\Source\Temp_Process\INTEL_01_v1_20231208_TELINV04_WIP.csv
2025-04-02 08:22:08.424 +05:30 [INF] Process file name :- INTEL_01_v1_20231208_TELINV04_WIP.csv
2025-04-02 08:22:08.424 +05:30 [INF] Total time taken in process: 32.9754829 seconds
2025-04-02 08:22:08.427 +05:30 [INF] Total time taken in moving: 0.002163 seconds
2025-04-02 08:22:14.700 +05:30 [INF] Application starting up
2025-04-02 08:22:15.627 +05:30 [INF] Main thread ID: 1
2025-04-02 08:22:15.628 +05:30 [INF] ARGS:- C:\BackOffice\ETL_Replacement\PIL\INTEL\Input\Source\Temp_Process\INTEL_01_v1_20231208_TELINV04_WIP.csv
2025-04-02 08:22:15.630 +05:30 [INF] Processing files for domain: PIL, invoiceType: INTEL
2025-04-02 08:22:17.093 +05:30 [INF] Exception in ProcessTaskAsync : Could not find file 'C:\BackOffice\ETL_Replacement\PIL\INTEL\Input\Source\Temp_Process\INTEL_01_v1_20231208_TELINV04_WIP.csv'.
2025-04-02 08:22:17.161 +05:30 [INF] Exception in ProcessFileAsync : Could not find file 'C:\BackOffice\ETL_Replacement\PIL\INTEL\Input\Source\Temp_Process\INTEL_01_v1_20231208_TELINV04_WIP.csv'.
2025-04-02 08:22:17.220 +05:30 [INF] Error processing files for domain PIL: Could not find file 'C:\BackOffice\ETL_Replacement\PIL\INTEL\Input\Source\Temp_Process\INTEL_01_v1_20231208_TELINV04_WIP.csv'.
2025-04-02 08:22:17.220 +05:30 [INF] The following files were not found and skipped:
2025-04-02 08:22:17.221 +05:30 [INF] C:\BackOffice\ETL_Replacement\PIL\INTEL\Input\Source\Temp_Process\INTEL_01_v1_20231208_TELINV04_WIP.csv
2025-04-02 08:22:39.508 +05:30 [INF] Application starting up
2025-04-02 08:22:40.163 +05:30 [INF] Main thread ID: 1
2025-04-02 08:22:40.163 +05:30 [INF] ARGS:- C:\BackOffice\ETL_Replacement\PIL\INTEL\Input\Source\Temp_Process\INTEL_01_v1_20231208_TELINV04_WIP.csv
2025-04-02 08:22:40.165 +05:30 [INF] Processing files for domain: PIL, invoiceType: INTEL
2025-04-02 08:22:43.143 +05:30 [INF] Process file path :- C:\BackOffice\ETL_Replacement\PIL\INTEL\Input\Source\Temp_Process\INTEL_01_v1_20231208_TELINV04_WIP.csv
2025-04-02 08:22:43.143 +05:30 [INF] Process file name :- INTEL_01_v1_20231208_TELINV04_WIP.csv
2025-04-02 08:22:43.144 +05:30 [INF] Total time taken in process: 1.5322091 seconds
2025-04-02 08:22:43.147 +05:30 [INF] Total time taken in moving: 0.0026288 seconds
2025-04-02 08:25:43.983 +05:30 [INF] Application starting up
2025-04-02 08:25:44.688 +05:30 [INF] Main thread ID: 1
2025-04-02 08:25:44.688 +05:30 [INF] ARGS:- C:\BackOffice\ETL_Replacement\PIL\INTEL\Input\Source\Temp_Process\INTEL_01_v1_20231208_TELINV04_WIP.csv
2025-04-02 08:25:44.691 +05:30 [INF] Processing files for domain: PIL, invoiceType: INTEL
2025-04-02 08:25:47.473 +05:30 [WRN] Exception In TempInsertStoreProc: P0003: query returned more than one row
2025-04-02 08:25:47.547 +05:30 [ERR] Connection failed. Retrying in 5 seconds... Attempt 1
2025-04-02 08:25:47.547 +05:30 [ERR] Exception: P0003: query returned more than one row
2025-04-02 08:25:52.713 +05:30 [WRN] Exception In TempInsertStoreProc: P0003: query returned more than one row
2025-04-02 08:25:52.759 +05:30 [ERR] Connection failed. Retrying in 5 seconds... Attempt 2
2025-04-02 08:25:52.759 +05:30 [ERR] Exception: P0003: query returned more than one row
2025-04-02 08:26:58.804 +05:30 [INF] Application starting up
2025-04-02 08:26:59.503 +05:30 [INF] Main thread ID: 1
2025-04-02 08:26:59.506 +05:30 [INF] ARGS:- C:\BackOffice\ETL_Replacement\PIL\INTEL\Input\Source\Temp_Process\INTEL_01_v1_20231208_TELINV04_WIP.csv
2025-04-02 08:26:59.510 +05:30 [INF] Processing files for domain: PIL, invoiceType: INTEL
2025-04-02 08:27:02.371 +05:30 [INF] Process file path :- C:\BackOffice\ETL_Replacement\PIL\INTEL\Input\Source\Temp_Process\INTEL_01_v1_20231208_TELINV04_WIP.csv
2025-04-02 08:27:02.371 +05:30 [INF] Process file name :- INTEL_01_v1_20231208_TELINV04_WIP.csv
2025-04-02 08:27:02.371 +05:30 [INF] Total time taken in process: 1.4819665 seconds
2025-04-02 08:27:02.374 +05:30 [INF] Total time taken in moving: 0.0026881 seconds
2025-04-02 08:31:56.756 +05:30 [INF] Application starting up
2025-04-02 08:31:57.986 +05:30 [INF] Main thread ID: 1
2025-04-02 08:31:58.002 +05:30 [INF] ARGS:- C:\BackOffice\ETL_Replacement\PIL\INTEL\Input\Source\Temp_Process\INTEL_01_v1_20231208_TELINV04_WIP.csv
2025-04-02 08:31:58.004 +05:30 [INF] Processing files for domain: PIL, invoiceType: INTEL
2025-04-02 08:32:00.894 +05:30 [WRN] Exception In TempInsertStoreProc: P0003: query returned more than one row
2025-04-02 08:32:00.970 +05:30 [ERR] Connection failed. Retrying in 5 seconds... Attempt 1
2025-04-02 08:32:00.970 +05:30 [ERR] Exception: P0003: query returned more than one row
2025-04-02 08:34:47.846 +05:30 [INF] Application starting up
2025-04-02 08:34:48.582 +05:30 [INF] Main thread ID: 1
2025-04-02 08:34:48.582 +05:30 [INF] ARGS:- C:\BackOffice\ETL_Replacement\PIL\INTEL\Input\Source\Temp_Process\INTEL_01_v1_20231208_TELINV04_WIP.csv
2025-04-02 08:34:48.584 +05:30 [INF] Processing files for domain: PIL, invoiceType: INTEL
2025-04-02 08:34:51.719 +05:30 [WRN] Exception In TempInsertStoreProc: 42703: column "invoice_line" does not exist
2025-04-02 08:34:51.793 +05:30 [ERR] Connection failed. Retrying in 5 seconds... Attempt 1
2025-04-02 08:34:51.793 +05:30 [ERR] Exception: 42703: column "invoice_line" does not exist
2025-04-02 08:36:01.840 +05:30 [INF] Application starting up
2025-04-02 08:36:02.539 +05:30 [INF] Main thread ID: 1
2025-04-02 08:36:02.539 +05:30 [INF] ARGS:- C:\BackOffice\ETL_Replacement\PIL\INTEL\Input\Source\Temp_Process\INTEL_01_v1_20231208_TELINV04_WIP.csv
2025-04-02 08:36:02.541 +05:30 [INF] Processing files for domain: PIL, invoiceType: INTEL
2025-04-02 08:36:05.148 +05:30 [INF] Process file path :- C:\BackOffice\ETL_Replacement\PIL\INTEL\Input\Source\Temp_Process\INTEL_01_v1_20231208_TELINV04_WIP.csv
2025-04-02 08:36:05.148 +05:30 [INF] Process file name :- INTEL_01_v1_20231208_TELINV04_WIP.csv
2025-04-02 08:36:05.149 +05:30 [INF] Total time taken in process: 1.2843927 seconds
2025-04-02 08:36:05.151 +05:30 [INF] Total time taken in moving: 0.0024647 seconds
2025-04-02 08:43:24.070 +05:30 [INF] Application starting up
2025-04-02 08:43:24.942 +05:30 [INF] Main thread ID: 1
2025-04-02 08:43:24.942 +05:30 [INF] ARGS:- C:\BackOffice\ETL_Replacement\PIL\INTEL\Input\Source\Temp_Process\INTEL_01_v1_20231208_TELINV04_WIP.csv
2025-04-02 08:43:24.944 +05:30 [INF] Processing files for domain: PIL, invoiceType: INTEL
2025-04-02 08:43:27.884 +05:30 [WRN] Exception In TempInsertStoreProc: P0003: query returned more than one row
2025-04-02 08:43:27.960 +05:30 [ERR] Connection failed. Retrying in 5 seconds... Attempt 1
2025-04-02 08:43:27.960 +05:30 [ERR] Exception: P0003: query returned more than one row
2025-04-02 08:44:21.675 +05:30 [INF] Application starting up
2025-04-02 08:44:22.423 +05:30 [INF] Main thread ID: 1
2025-04-02 08:44:22.424 +05:30 [INF] ARGS:- C:\BackOffice\ETL_Replacement\PIL\INTEL\Input\Source\Temp_Process\INTEL_01_v1_20231208_TELINV04_WIP.csv
2025-04-02 08:44:22.426 +05:30 [INF] Processing files for domain: PIL, invoiceType: INTEL
2025-04-02 08:44:25.403 +05:30 [WRN] Exception In TempInsertStoreProc: P0003: query returned more than one row
2025-04-02 08:44:25.477 +05:30 [ERR] Connection failed. Retrying in 5 seconds... Attempt 1
2025-04-02 08:44:25.477 +05:30 [ERR] Exception: P0003: query returned more than one row
2025-04-02 08:46:25.052 +05:30 [INF] Application starting up
2025-04-02 08:46:25.848 +05:30 [INF] Main thread ID: 1
2025-04-02 08:46:25.849 +05:30 [INF] ARGS:- C:\BackOffice\ETL_Replacement\PIL\INTEL\Input\Source\Temp_Process\INTEL_01_v1_20231208_TELINV04_WIP.csv
2025-04-02 08:46:25.851 +05:30 [INF] Processing files for domain: PIL, invoiceType: INTEL
2025-04-02 08:46:30.179 +05:30 [INF] Process file path :- C:\BackOffice\ETL_Replacement\PIL\INTEL\Input\Source\Temp_Process\INTEL_01_v1_20231208_TELINV04_WIP.csv
2025-04-02 08:46:30.179 +05:30 [INF] Process file name :- INTEL_01_v1_20231208_TELINV04_WIP.csv
2025-04-02 08:46:30.179 +05:30 [INF] Total time taken in process: 1.3583356 seconds
2025-04-02 08:46:30.182 +05:30 [INF] Total time taken in moving: 0.0025216 seconds
2025-04-02 08:48:58.724 +05:30 [INF] Application starting up
2025-04-02 08:48:59.753 +05:30 [INF] Main thread ID: 1
2025-04-02 08:48:59.754 +05:30 [INF] ARGS:- C:\BackOffice\ETL_Replacement\PIL\INTEL\Input\Source\Temp_Process\INTEL_01_v1_20231208_TELINV04_WIP.csv
2025-04-02 08:48:59.757 +05:30 [INF] Processing files for domain: PIL, invoiceType: INTEL
2025-04-02 08:49:02.820 +05:30 [WRN] Exception In TempInsertStoreProc: P0003: query returned more than one row
2025-04-02 08:49:02.891 +05:30 [ERR] Connection failed. Retrying in 5 seconds... Attempt 1
2025-04-02 08:49:02.891 +05:30 [ERR] Exception: P0003: query returned more than one row
2025-04-02 08:52:24.578 +05:30 [INF] Application starting up
2025-04-02 08:52:25.323 +05:30 [INF] Main thread ID: 1
2025-04-02 08:52:25.323 +05:30 [INF] ARGS:- C:\BackOffice\ETL_Replacement\PIL\INTEL\Input\Source\Temp_Process\INTEL_01_v1_20231208_TELINV04_WIP.csv
2025-04-02 08:52:25.325 +05:30 [INF] Processing files for domain: PIL, invoiceType: INTEL
2025-04-02 08:52:47.861 +05:30 [INF] Process file path :- C:\BackOffice\ETL_Replacement\PIL\INTEL\Input\Source\Temp_Process\INTEL_01_v1_20231208_TELINV04_WIP.csv
2025-04-02 08:52:47.861 +05:30 [INF] Process file name :- INTEL_01_v1_20231208_TELINV04_WIP.csv
2025-04-02 08:52:47.862 +05:30 [INF] Total time taken in process: 18.1161452 seconds
2025-04-02 08:52:47.864 +05:30 [INF] Total time taken in moving: 0.0024975 seconds
2025-04-02 08:54:42.937 +05:30 [INF] Application starting up
2025-04-02 08:54:43.688 +05:30 [INF] Main thread ID: 1
2025-04-02 08:54:43.688 +05:30 [INF] ARGS:- C:\BackOffice\ETL_Replacement\PIL\INTEL\Input\Source\Temp_Process\INTEL_01_v1_20231208_TELINV04_WIP.csv
2025-04-02 08:54:43.692 +05:30 [INF] Processing files for domain: PIL, invoiceType: INTEL
2025-04-02 08:54:56.522 +05:30 [WRN] Exception In TempInsertStoreProc: 42P01: missing FROM-clause entry for table "line_item"
2025-04-02 08:54:56.596 +05:30 [ERR] Connection failed. Retrying in 5 seconds... Attempt 1
2025-04-02 08:54:56.596 +05:30 [ERR] Exception: 42P01: missing FROM-clause entry for table "line_item"
2025-04-02 08:59:09.405 +05:30 [INF] Application starting up
2025-04-02 08:59:10.141 +05:30 [INF] Main thread ID: 1
2025-04-02 08:59:10.141 +05:30 [INF] ARGS:- C:\BackOffice\ETL_Replacement\PIL\INTEL\Input\Source\Temp_Process\INTEL_01_v1_20231208_TELINV04_WIP.csv
2025-04-02 08:59:10.144 +05:30 [INF] Processing files for domain: PIL, invoiceType: INTEL
2025-04-02 08:59:22.201 +05:30 [WRN] Exception In TempInsertStoreProc: 42601: query has no destination for result data
2025-04-02 08:59:22.273 +05:30 [ERR] Connection failed. Retrying in 5 seconds... Attempt 1
2025-04-02 08:59:22.273 +05:30 [ERR] Exception: 42601: query has no destination for result data
2025-04-02 08:59:28.066 +05:30 [WRN] Exception In TempInsertStoreProc: 42601: query has no destination for result data
2025-04-02 08:59:28.112 +05:30 [ERR] Connection failed. Retrying in 5 seconds... Attempt 2
2025-04-02 08:59:28.112 +05:30 [ERR] Exception: 42601: query has no destination for result data
2025-04-02 09:03:18.905 +05:30 [INF] Application starting up
2025-04-02 09:03:19.722 +05:30 [INF] Main thread ID: 1
2025-04-02 09:03:19.724 +05:30 [INF] ARGS:- C:\BackOffice\ETL_Replacement\PIL\INTEL\Input\Source\Temp_Process\INTEL_01_v1_20231208_TELINV04_WIP.csv
2025-04-02 09:03:19.728 +05:30 [INF] Processing files for domain: PIL, invoiceType: INTEL
2025-04-02 09:03:26.126 +05:30 [INF] Process file path :- C:\BackOffice\ETL_Replacement\PIL\INTEL\Input\Source\Temp_Process\INTEL_01_v1_20231208_TELINV04_WIP.csv
2025-04-02 09:03:26.127 +05:30 [INF] Process file name :- INTEL_01_v1_20231208_TELINV04_WIP.csv
2025-04-02 09:03:26.127 +05:30 [INF] Total time taken in process: 2.0410797 seconds
2025-04-02 09:03:26.130 +05:30 [INF] Total time taken in moving: 0.0027128 seconds
2025-04-02 09:05:56.515 +05:30 [INF] Application starting up
2025-04-02 09:05:57.166 +05:30 [INF] Main thread ID: 1
2025-04-02 09:05:57.166 +05:30 [INF] ARGS:- C:\BackOffice\ETL_Replacement\PIL\INTEL\Input\Source\Temp_Process\INTEL_01_v1_20231208_TELINV04_WIP.csv
2025-04-02 09:05:57.169 +05:30 [INF] Processing files for domain: PIL, invoiceType: INTEL
2025-04-02 09:06:00.529 +05:30 [WRN] Exception In TempInsertStoreProc: P0001: pdf_fields BGL24038088
2025-04-02 09:06:00.606 +05:30 [ERR] Connection failed. Retrying in 5 seconds... Attempt 1
2025-04-02 09:06:00.606 +05:30 [ERR] Exception: P0001: pdf_fields BGL24038088
2025-04-02 09:06:05.785 +05:30 [WRN] Exception In TempInsertStoreProc: P0001: pdf_fields BGL24038088
2025-04-02 09:06:05.834 +05:30 [ERR] Connection failed. Retrying in 5 seconds... Attempt 2
2025-04-02 09:06:05.834 +05:30 [ERR] Exception: P0001: pdf_fields BGL24038088
2025-04-02 09:06:11.014 +05:30 [WRN] Exception In TempInsertStoreProc: P0001: pdf_fields BGL24038088
2025-04-02 09:06:11.060 +05:30 [ERR] Connection failed. Retrying in 5 seconds... Attempt 3
2025-04-02 09:06:11.060 +05:30 [ERR] Exception: P0001: pdf_fields BGL24038088
2025-04-02 09:06:30.269 +05:30 [INF] Application starting up
2025-04-02 09:06:31.327 +05:30 [INF] Main thread ID: 1
2025-04-02 09:06:31.327 +05:30 [INF] ARGS:- C:\BackOffice\ETL_Replacement\PIL\INTEL\Input\Source\Temp_Process\INTEL_01_v1_20231208_TELINV04_WIP.csv
2025-04-02 09:06:31.330 +05:30 [INF] Processing files for domain: PIL, invoiceType: INTEL
2025-04-02 09:06:34.153 +05:30 [WRN] Exception In TempInsertStoreProc: P0001: pdf_fields <NULL>
2025-04-02 09:06:34.228 +05:30 [ERR] Connection failed. Retrying in 5 seconds... Attempt 1
2025-04-02 09:06:34.228 +05:30 [ERR] Exception: P0001: pdf_fields <NULL>
2025-04-02 09:08:03.168 +05:30 [INF] Application starting up
2025-04-02 09:08:04.038 +05:30 [INF] Main thread ID: 1
2025-04-02 09:08:04.044 +05:30 [INF] ARGS:- C:\BackOffice\ETL_Replacement\PIL\INTEL\Input\Source\Temp_Process\INTEL_01_v1_20231208_TELINV04_WIP.csv
2025-04-02 09:08:04.049 +05:30 [INF] Processing files for domain: PIL, invoiceType: INTEL
2025-04-02 09:08:12.086 +05:30 [INF] Application starting up
2025-04-02 09:08:12.825 +05:30 [INF] Main thread ID: 1
2025-04-02 09:08:12.826 +05:30 [INF] ARGS:- C:\BackOffice\ETL_Replacement\PIL\INTEL\Input\Source\Temp_Process\INTEL_01_v1_20231208_TELINV04_WIP.csv
2025-04-02 09:08:12.827 +05:30 [INF] Processing files for domain: PIL, invoiceType: INTEL
2025-04-02 09:08:15.525 +05:30 [WRN] Exception In TempInsertStoreProc: P0001: pdf_fields <NULL>
2025-04-02 09:08:15.601 +05:30 [ERR] Connection failed. Retrying in 5 seconds... Attempt 1
2025-04-02 09:08:15.601 +05:30 [ERR] Exception: P0001: pdf_fields <NULL>
2025-04-02 09:08:20.777 +05:30 [WRN] Exception In TempInsertStoreProc: P0001: pdf_fields <NULL>
2025-04-02 09:09:12.438 +05:30 [INF] Application starting up
2025-04-02 09:09:13.375 +05:30 [INF] Main thread ID: 1
2025-04-02 09:09:13.376 +05:30 [INF] ARGS:- C:\BackOffice\ETL_Replacement\PIL\INTEL\Input\Source\Temp_Process\INTEL_01_v1_20231208_TELINV04_WIP.csv
2025-04-02 09:09:13.378 +05:30 [INF] Processing files for domain: PIL, invoiceType: INTEL
2025-04-02 09:09:16.151 +05:30 [WRN] Exception In TempInsertStoreProc: P0001: pdf_fields <NULL>
2025-04-02 09:09:16.233 +05:30 [ERR] Connection failed. Retrying in 5 seconds... Attempt 1
2025-04-02 09:09:16.233 +05:30 [ERR] Exception: P0001: pdf_fields <NULL>
2025-04-02 09:09:21.426 +05:30 [WRN] Exception In TempInsertStoreProc: P0001: pdf_fields <NULL>
2025-04-02 09:11:32.130 +05:30 [INF] Application starting up
2025-04-02 09:11:33.000 +05:30 [INF] Main thread ID: 1
2025-04-02 09:11:33.003 +05:30 [INF] ARGS:- C:\BackOffice\ETL_Replacement\PIL\INTEL\Input\Source\Temp_Process\INTEL_01_v1_20231208_TELINV04_WIP.csv
2025-04-02 09:11:33.005 +05:30 [INF] Processing files for domain: PIL, invoiceType: INTEL
2025-04-02 09:11:35.770 +05:30 [WRN] Exception In TempInsertStoreProc: P0001: pdf_fields <NULL>
2025-04-02 09:11:35.843 +05:30 [ERR] Connection failed. Retrying in 5 seconds... Attempt 1
2025-04-02 09:11:35.843 +05:30 [ERR] Exception: P0001: pdf_fields <NULL>
2025-04-02 09:16:04.589 +05:30 [INF] Application starting up
2025-04-02 09:16:05.534 +05:30 [INF] Main thread ID: 1
2025-04-02 09:16:05.534 +05:30 [INF] ARGS:- C:\BackOffice\ETL_Replacement\PIL\INTEL\Input\Source\Temp_Process\INTEL_01_v1_20231208_TELINV04_WIP.csv
2025-04-02 09:16:05.537 +05:30 [INF] Processing files for domain: PIL, invoiceType: INTEL
2025-04-02 09:16:08.292 +05:30 [WRN] Exception In TempInsertStoreProc: P0001: pdf_fields 3237
2025-04-02 09:16:08.367 +05:30 [ERR] Connection failed. Retrying in 5 seconds... Attempt 1
2025-04-02 09:16:08.367 +05:30 [ERR] Exception: P0001: pdf_fields 3237
2025-04-02 09:16:13.535 +05:30 [WRN] Exception In TempInsertStoreProc: P0001: pdf_fields 3237
2025-04-02 09:16:13.585 +05:30 [ERR] Connection failed. Retrying in 5 seconds... Attempt 2
2025-04-02 09:16:13.586 +05:30 [ERR] Exception: P0001: pdf_fields 3237
2025-04-02 09:16:18.742 +05:30 [WRN] Exception In TempInsertStoreProc: P0001: pdf_fields 3237
2025-04-02 09:16:18.793 +05:30 [ERR] Connection failed. Retrying in 5 seconds... Attempt 3
2025-04-02 09:16:18.794 +05:30 [ERR] Exception: P0001: pdf_fields 3237
2025-04-02 09:16:24.036 +05:30 [WRN] Exception In TempInsertStoreProc: P0001: pdf_fields 3237
2025-04-02 09:16:24.090 +05:30 [ERR] Connection failed. Retrying in 5 seconds... Attempt 4
2025-04-02 09:16:24.090 +05:30 [ERR] Exception: P0001: pdf_fields 3237
2025-04-02 09:16:29.244 +05:30 [WRN] Exception In TempInsertStoreProc: P0001: pdf_fields 3237
2025-04-02 09:16:29.291 +05:30 [ERR] Connection failed. Retrying in 5 seconds... Attempt 5
2025-04-02 09:16:29.292 +05:30 [ERR] Exception: P0001: pdf_fields 3237
2025-04-02 09:16:34.456 +05:30 [WRN] Exception In TempInsertStoreProc: P0001: pdf_fields 3237
2025-04-02 09:16:34.505 +05:30 [ERR] Connection failed. Retrying in 5 seconds... Attempt 6
2025-04-02 09:16:34.506 +05:30 [ERR] Exception: P0001: pdf_fields 3237
2025-04-02 09:16:39.735 +05:30 [INF] Process file path :- C:\BackOffice\ETL_Replacement\PIL\INTEL\Input\Source\Temp_Process\INTEL_01_v1_20231208_TELINV04_WIP.csv
2025-04-02 09:16:39.735 +05:30 [INF] Process file name :- INTEL_01_v1_20231208_TELINV04_WIP.csv
2025-04-02 09:16:39.735 +05:30 [INF] Total time taken in process: 32.7734518 seconds
2025-04-02 09:16:39.738 +05:30 [INF] Total time taken in moving: 0.0021727 seconds
2025-04-02 09:16:54.621 +05:30 [INF] Application starting up
2025-04-02 09:16:55.938 +05:30 [INF] Main thread ID: 1
2025-04-02 09:16:55.956 +05:30 [INF] ARGS:- C:\BackOffice\ETL_Replacement\PIL\INTEL\Input\Source\Temp_Process\INTEL_01_v1_20231208_TELINV04_WIP.csv
2025-04-02 09:16:55.972 +05:30 [INF] Processing files for domain: PIL, invoiceType: INTEL
2025-04-02 09:16:58.745 +05:30 [INF] Exception in ProcessTaskAsync : Could not find file 'C:\BackOffice\ETL_Replacement\PIL\INTEL\Input\Source\Temp_Process\INTEL_01_v1_20231208_TELINV04_WIP.csv'.
2025-04-02 09:16:58.816 +05:30 [INF] Exception in ProcessFileAsync : Could not find file 'C:\BackOffice\ETL_Replacement\PIL\INTEL\Input\Source\Temp_Process\INTEL_01_v1_20231208_TELINV04_WIP.csv'.
2025-04-02 09:16:58.876 +05:30 [INF] Error processing files for domain PIL: Could not find file 'C:\BackOffice\ETL_Replacement\PIL\INTEL\Input\Source\Temp_Process\INTEL_01_v1_20231208_TELINV04_WIP.csv'.
2025-04-02 09:16:58.876 +05:30 [INF] The following files were not found and skipped:
2025-04-02 09:16:58.876 +05:30 [INF] C:\BackOffice\ETL_Replacement\PIL\INTEL\Input\Source\Temp_Process\INTEL_01_v1_20231208_TELINV04_WIP.csv
2025-04-02 09:17:21.365 +05:30 [INF] Application starting up
2025-04-02 09:17:22.120 +05:30 [INF] Main thread ID: 1
2025-04-02 09:17:22.121 +05:30 [INF] ARGS:- C:\BackOffice\ETL_Replacement\PIL\INTEL\Input\Source\Temp_Process\INTEL_01_v1_20231208_TELINV04_WIP.csv
2025-04-02 09:17:22.123 +05:30 [INF] Processing files for domain: PIL, invoiceType: INTEL
2025-04-02 09:17:33.774 +05:30 [INF] Process file path :- C:\BackOffice\ETL_Replacement\PIL\INTEL\Input\Source\Temp_Process\INTEL_01_v1_20231208_TELINV04_WIP.csv
2025-04-02 09:17:33.775 +05:30 [INF] Process file name :- INTEL_01_v1_20231208_TELINV04_WIP.csv
2025-04-02 09:17:33.775 +05:30 [INF] Total time taken in process: 10.2203975 seconds
2025-04-02 09:17:33.778 +05:30 [INF] Total time taken in moving: 0.0024713 seconds
2025-04-02 09:20:08.579 +05:30 [INF] Application starting up
2025-04-02 09:20:09.429 +05:30 [INF] Main thread ID: 1
2025-04-02 09:20:09.429 +05:30 [INF] ARGS:- C:\BackOffice\ETL_Replacement\PIL\INTEL\Input\Source\Temp_Process\INTEL_01_v1_20231208_TELINV04_WIP.csv
2025-04-02 09:20:09.432 +05:30 [INF] Processing files for domain: PIL, invoiceType: INTEL
2025-04-02 09:21:32.623 +05:30 [INF] Application starting up
2025-04-02 09:21:33.622 +05:30 [INF] Main thread ID: 1
2025-04-02 09:21:33.622 +05:30 [INF] ARGS:- C:\BackOffice\ETL_Replacement\PIL\INTEL\Input\Source\Temp_Process\INTEL_01_v1_20231208_TELINV04_WIP.csv
2025-04-02 09:21:33.625 +05:30 [INF] Processing files for domain: PIL, invoiceType: INTEL
2025-04-02 09:24:10.102 +05:30 [INF] Application starting up
2025-04-02 09:24:11.211 +05:30 [INF] Main thread ID: 1
2025-04-02 09:24:11.211 +05:30 [INF] ARGS:- C:\BackOffice\ETL_Replacement\PIL\INTEL\Input\Source\Temp_Process\INTEL_01_v1_20231208_TELINV04_WIP.csv
2025-04-02 09:24:11.214 +05:30 [INF] Processing files for domain: PIL, invoiceType: INTEL
2025-04-02 09:24:15.327 +05:30 [WRN] Exception In TempInsertStoreProc: 42804: column "settlingdate" is of type timestamp without time zone but expression is of type text
2025-04-02 09:24:15.403 +05:30 [ERR] Connection failed. Retrying in 5 seconds... Attempt 1
2025-04-02 09:24:15.403 +05:30 [ERR] Exception: 42804: column "settlingdate" is of type timestamp without time zone but expression is of type text
2025-04-02 09:25:15.172 +05:30 [INF] Application starting up
2025-04-02 09:25:16.016 +05:30 [INF] Main thread ID: 1
2025-04-02 09:25:16.016 +05:30 [INF] ARGS:- C:\BackOffice\ETL_Replacement\PIL\INTEL\Input\Source\Temp_Process\INTEL_01_v1_20231208_TELINV04_WIP.csv
2025-04-02 09:25:16.018 +05:30 [INF] Processing files for domain: PIL, invoiceType: INTEL
2025-04-02 09:25:18.875 +05:30 [INF] Process file path :- C:\BackOffice\ETL_Replacement\PIL\INTEL\Input\Source\Temp_Process\INTEL_01_v1_20231208_TELINV04_WIP.csv
2025-04-02 09:25:18.875 +05:30 [INF] Process file name :- INTEL_01_v1_20231208_TELINV04_WIP.csv
2025-04-02 09:25:18.876 +05:30 [INF] Total time taken in process: 1.5423231 seconds
2025-04-02 09:25:18.879 +05:30 [INF] Total time taken in moving: 0.0024783 seconds
2025-04-02 09:31:23.475 +05:30 [INF] Application starting up
2025-04-02 09:31:24.497 +05:30 [INF] Main thread ID: 1
2025-04-02 09:31:24.497 +05:30 [INF] ARGS:- C:\BackOffice\ETL_Replacement\PIL\INTEL\Input\Source\Temp_Process\INTEL_01_v1_20231208_TELINV04_WIP.csv
2025-04-02 09:31:24.500 +05:30 [INF] Processing files for domain: PIL, invoiceType: INTEL
2025-04-02 09:31:27.431 +05:30 [INF] Process file path :- C:\BackOffice\ETL_Replacement\PIL\INTEL\Input\Source\Temp_Process\INTEL_01_v1_20231208_TELINV04_WIP.csv
2025-04-02 09:31:27.431 +05:30 [INF] Process file name :- INTEL_01_v1_20231208_TELINV04_WIP.csv
2025-04-02 09:31:27.432 +05:30 [INF] Total time taken in process: 1.5583345 seconds
2025-04-02 09:31:27.435 +05:30 [INF] Total time taken in moving: 0.0025295 seconds
2025-04-02 09:32:28.660 +05:30 [INF] Application starting up
2025-04-02 09:32:30.199 +05:30 [INF] Main thread ID: 1
2025-04-02 09:32:30.199 +05:30 [INF] ARGS:- C:\BackOffice\ETL_Replacement\PIL\INTEL\Input\Source\Temp_Process\INTEL_01_v1_20231208_TELINV04_WIP.csv
2025-04-02 09:32:30.204 +05:30 [INF] Processing files for domain: PIL, invoiceType: INTEL
2025-04-02 09:32:33.102 +05:30 [INF] Process file path :- C:\BackOffice\ETL_Replacement\PIL\INTEL\Input\Source\Temp_Process\INTEL_01_v1_20231208_TELINV04_WIP.csv
2025-04-02 09:32:33.102 +05:30 [INF] Process file name :- INTEL_01_v1_20231208_TELINV04_WIP.csv
2025-04-02 09:32:33.102 +05:30 [INF] Total time taken in process: 1.5961097 seconds
2025-04-02 09:32:33.105 +05:30 [INF] Total time taken in moving: 0.0026436 seconds
2025-04-02 09:45:52.480 +05:30 [INF] Application starting up
2025-04-02 09:45:53.179 +05:30 [INF] Main thread ID: 1
2025-04-02 09:45:53.179 +05:30 [INF] ARGS:- C:\BackOffice\ETL_Replacement\PIL\INTEL\Input\Source\Temp_Process\INTEL_01_v1_20231208_TELINV04_WIP.csv
2025-04-02 09:45:53.182 +05:30 [INF] Processing files for domain: PIL, invoiceType: INTEL
2025-04-02 09:45:59.203 +05:30 [ERR] Connection failed. Retrying in 5 seconds... Attempt 1
2025-04-02 09:45:59.203 +05:30 [ERR] Exception: 42703: column "accountnumber" does not exist
2025-04-02 09:47:37.720 +05:30 [INF] Application starting up
2025-04-02 09:47:38.594 +05:30 [INF] Main thread ID: 1
2025-04-02 09:47:38.595 +05:30 [INF] ARGS:- C:\BackOffice\ETL_Replacement\PIL\INTEL\Input\Source\Temp_Process\INTEL_01_v1_20231208_TELINV04_WIP.csv
2025-04-02 09:47:38.600 +05:30 [INF] Processing files for domain: PIL, invoiceType: INTEL
2025-04-02 09:47:42.045 +05:30 [ERR] Connection failed. Retrying in 5 seconds... Attempt 1
2025-04-02 09:47:42.045 +05:30 [ERR] Exception: 42P01: relation "inserted_invoices" does not exist
2025-04-02 09:49:12.887 +05:30 [INF] Application starting up
2025-04-02 09:49:13.908 +05:30 [INF] Main thread ID: 1
2025-04-02 09:49:13.908 +05:30 [INF] ARGS:- C:\BackOffice\ETL_Replacement\PIL\INTEL\Input\Source\Temp_Process\INTEL_01_v1_20231208_TELINV04_WIP.csv
2025-04-02 09:49:13.910 +05:30 [INF] Processing files for domain: PIL, invoiceType: INTEL
2025-04-02 09:49:17.022 +05:30 [ERR] Connection failed. Retrying in 5 seconds... Attempt 1
2025-04-02 09:49:17.022 +05:30 [ERR] Exception: 42P01: relation "inserted_invoices" does not exist
2025-04-02 09:49:40.936 +05:30 [INF] Application starting up
2025-04-02 09:49:41.834 +05:30 [INF] Main thread ID: 1
2025-04-02 09:49:41.834 +05:30 [INF] ARGS:- C:\BackOffice\ETL_Replacement\PIL\INTEL\Input\Source\Temp_Process\INTEL_01_v1_20231208_TELINV04_WIP.csv
2025-04-02 09:49:41.836 +05:30 [INF] Processing files for domain: PIL, invoiceType: INTEL
2025-04-02 09:49:45.058 +05:30 [ERR] Connection failed. Retrying in 5 seconds... Attempt 1
2025-04-02 09:49:45.058 +05:30 [ERR] Exception: 42P01: relation "inserted_invoices" does not exist
2025-04-02 09:52:14.943 +05:30 [INF] Application starting up
2025-04-02 09:52:15.654 +05:30 [INF] Main thread ID: 1
2025-04-02 09:52:15.654 +05:30 [INF] ARGS:- C:\BackOffice\ETL_Replacement\PIL\INTEL\Input\Source\Temp_Process\INTEL_01_v1_20231208_TELINV04_WIP.csv
2025-04-02 09:52:15.657 +05:30 [INF] Processing files for domain: PIL, invoiceType: INTEL
2025-04-02 09:52:19.834 +05:30 [ERR] Connection failed. Retrying in 5 seconds... Attempt 1
2025-04-02 09:52:19.834 +05:30 [ERR] Exception: 42P01: relation "inserted_invoices" does not exist
2025-04-02 09:52:25.217 +05:30 [ERR] Connection failed. Retrying in 5 seconds... Attempt 2
2025-04-02 09:52:25.217 +05:30 [ERR] Exception: 42P01: relation "inserted_invoices" does not exist
2025-04-02 09:52:30.563 +05:30 [ERR] Connection failed. Retrying in 5 seconds... Attempt 3
2025-04-02 09:52:30.563 +05:30 [ERR] Exception: 42P01: relation "inserted_invoices" does not exist
2025-04-02 09:52:35.963 +05:30 [ERR] Connection failed. Retrying in 5 seconds... Attempt 4
2025-04-02 09:52:35.963 +05:30 [ERR] Exception: 42P01: relation "inserted_invoices" does not exist
2025-04-02 09:52:41.344 +05:30 [ERR] Connection failed. Retrying in 5 seconds... Attempt 5
2025-04-02 09:52:41.346 +05:30 [ERR] Exception: 42P01: relation "inserted_invoices" does not exist
2025-04-02 09:52:46.695 +05:30 [ERR] Connection failed. Retrying in 5 seconds... Attempt 6
2025-04-02 09:52:46.696 +05:30 [ERR] Exception: 42P01: relation "inserted_invoices" does not exist
2025-04-02 09:52:52.112 +05:30 [ERR] Connection failed. Retrying in 5 seconds... Attempt 7
2025-04-02 09:52:52.113 +05:30 [ERR] Exception: 42P01: relation "inserted_invoices" does not exist
2025-04-02 09:52:57.438 +05:30 [ERR] Connection failed. Retrying in 5 seconds... Attempt 8
2025-04-02 09:52:57.438 +05:30 [ERR] Exception: 42P01: relation "inserted_invoices" does not exist
2025-04-02 09:53:02.793 +05:30 [ERR] Connection failed. Retrying in 5 seconds... Attempt 9
2025-04-02 09:53:02.794 +05:30 [ERR] Exception: 42P01: relation "inserted_invoices" does not exist
2025-04-02 09:53:08.558 +05:30 [ERR] Connection failed. Retrying in 5 seconds... Attempt 10
2025-04-02 09:53:08.559 +05:30 [ERR] Exception: 42P01: relation "inserted_invoices" does not exist
2025-04-02 09:53:13.919 +05:30 [ERR] Connection failed. Retrying in 5 seconds... Attempt 11
2025-04-02 09:53:13.920 +05:30 [ERR] Exception: 42P01: relation "inserted_invoices" does not exist
2025-04-02 09:53:19.366 +05:30 [ERR] Connection failed. Retrying in 5 seconds... Attempt 12
2025-04-02 09:53:19.366 +05:30 [ERR] Exception: 42P01: relation "inserted_invoices" does not exist
2025-04-02 09:53:24.730 +05:30 [ERR] Connection failed. Retrying in 5 seconds... Attempt 13
2025-04-02 09:53:24.731 +05:30 [ERR] Exception: 42P01: relation "inserted_invoices" does not exist
2025-04-02 09:53:30.086 +05:30 [ERR] Connection failed. Retrying in 5 seconds... Attempt 14
2025-04-02 09:53:30.087 +05:30 [ERR] Exception: 42P01: relation "inserted_invoices" does not exist
2025-04-02 09:53:35.459 +05:30 [ERR] Connection failed. Retrying in 5 seconds... Attempt 15
2025-04-02 09:53:35.460 +05:30 [ERR] Exception: 42P01: relation "inserted_invoices" does not exist
2025-04-02 09:53:40.903 +05:30 [ERR] Connection failed. Retrying in 5 seconds... Attempt 16
2025-04-02 09:53:40.904 +05:30 [ERR] Exception: 42P01: relation "inserted_invoices" does not exist
2025-04-02 09:53:46.274 +05:30 [ERR] Connection failed. Retrying in 5 seconds... Attempt 17
2025-04-02 09:53:46.274 +05:30 [ERR] Exception: 42P01: relation "inserted_invoices" does not exist
2025-04-02 09:53:51.589 +05:30 [ERR] Connection failed. Retrying in 5 seconds... Attempt 18
2025-04-02 09:53:51.590 +05:30 [ERR] Exception: 42P01: relation "inserted_invoices" does not exist
2025-04-02 09:53:57.177 +05:30 [ERR] Connection failed. Retrying in 5 seconds... Attempt 19
2025-04-02 09:53:57.177 +05:30 [ERR] Exception: 42P01: relation "inserted_invoices" does not exist
2025-04-02 09:54:02.556 +05:30 [ERR] Connection failed. Retrying in 5 seconds... Attempt 20
2025-04-02 09:54:02.556 +05:30 [ERR] Exception: 42P01: relation "inserted_invoices" does not exist
2025-04-02 09:54:08.155 +05:30 [ERR] Connection failed. Retrying in 5 seconds... Attempt 21
2025-04-02 09:54:08.155 +05:30 [ERR] Exception: 42P01: relation "inserted_invoices" does not exist
2025-04-02 09:54:13.461 +05:30 [ERR] Connection failed. Retrying in 5 seconds... Attempt 22
2025-04-02 09:54:13.462 +05:30 [ERR] Exception: 42P01: relation "inserted_invoices" does not exist
2025-04-02 09:54:18.911 +05:30 [ERR] Connection failed. Retrying in 5 seconds... Attempt 23
2025-04-02 09:54:18.911 +05:30 [ERR] Exception: 42P01: relation "inserted_invoices" does not exist
2025-04-02 09:54:24.349 +05:30 [ERR] Connection failed. Retrying in 5 seconds... Attempt 24
2025-04-02 09:54:24.350 +05:30 [ERR] Exception: 42P01: relation "inserted_invoices" does not exist
2025-04-02 09:54:29.644 +05:30 [ERR] Connection failed. Retrying in 5 seconds... Attempt 25
2025-04-02 09:54:29.645 +05:30 [ERR] Exception: 42P01: relation "inserted_invoices" does not exist
2025-04-02 09:54:35.008 +05:30 [ERR] Connection failed. Retrying in 5 seconds... Attempt 26
2025-04-02 09:54:35.008 +05:30 [ERR] Exception: 42P01: relation "inserted_invoices" does not exist
2025-04-02 09:54:40.400 +05:30 [ERR] Connection failed. Retrying in 5 seconds... Attempt 27
2025-04-02 09:54:40.400 +05:30 [ERR] Exception: 42P01: relation "inserted_invoices" does not exist
2025-04-02 09:54:45.819 +05:30 [ERR] Connection failed. Retrying in 5 seconds... Attempt 28
2025-04-02 09:54:45.820 +05:30 [ERR] Exception: 42P01: relation "inserted_invoices" does not exist
2025-04-02 09:54:51.357 +05:30 [ERR] Connection failed. Retrying in 5 seconds... Attempt 29
2025-04-02 09:54:51.358 +05:30 [ERR] Exception: 42P01: relation "inserted_invoices" does not exist
2025-04-02 09:54:56.743 +05:30 [ERR] Connection failed. Retrying in 5 seconds... Attempt 30
2025-04-02 09:54:56.744 +05:30 [ERR] Exception: 42P01: relation "inserted_invoices" does not exist
2025-04-02 09:55:02.071 +05:30 [ERR] Connection failed. Retrying in 5 seconds... Attempt 31
2025-04-02 09:55:02.072 +05:30 [ERR] Exception: 42P01: relation "inserted_invoices" does not exist
2025-04-02 09:55:07.481 +05:30 [ERR] Connection failed. Retrying in 5 seconds... Attempt 32
2025-04-02 09:55:07.482 +05:30 [ERR] Exception: 42P01: relation "inserted_invoices" does not exist
2025-04-02 09:55:12.797 +05:30 [ERR] Connection failed. Retrying in 5 seconds... Attempt 33
2025-04-02 09:55:12.798 +05:30 [ERR] Exception: 42P01: relation "inserted_invoices" does not exist
2025-04-02 09:55:18.186 +05:30 [ERR] Connection failed. Retrying in 5 seconds... Attempt 34
2025-04-02 09:55:18.187 +05:30 [ERR] Exception: 42P01: relation "inserted_invoices" does not exist
2025-04-02 09:55:23.586 +05:30 [ERR] Connection failed. Retrying in 5 seconds... Attempt 35
2025-04-02 09:55:23.586 +05:30 [ERR] Exception: 42P01: relation "inserted_invoices" does not exist
2025-04-02 09:55:29.036 +05:30 [ERR] Connection failed. Retrying in 5 seconds... Attempt 36
2025-04-02 09:55:29.036 +05:30 [ERR] Exception: 42P01: relation "inserted_invoices" does not exist
2025-04-02 09:55:34.462 +05:30 [ERR] Connection failed. Retrying in 5 seconds... Attempt 37
2025-04-02 09:55:34.463 +05:30 [ERR] Exception: 42P01: relation "inserted_invoices" does not exist
2025-04-02 09:55:39.902 +05:30 [ERR] Connection failed. Retrying in 5 seconds... Attempt 38
2025-04-02 09:55:39.903 +05:30 [ERR] Exception: 42P01: relation "inserted_invoices" does not exist
2025-04-02 09:55:45.308 +05:30 [ERR] Connection failed. Retrying in 5 seconds... Attempt 39
2025-04-02 09:55:45.308 +05:30 [ERR] Exception: 42P01: relation "inserted_invoices" does not exist
2025-04-02 09:55:50.743 +05:30 [ERR] Connection failed. Retrying in 5 seconds... Attempt 40
2025-04-02 09:55:50.743 +05:30 [ERR] Exception: 42P01: relation "inserted_invoices" does not exist
2025-04-02 09:55:56.098 +05:30 [ERR] Connection failed. Retrying in 5 seconds... Attempt 41
2025-04-02 09:55:56.099 +05:30 [ERR] Exception: 42P01: relation "inserted_invoices" does not exist
2025-04-02 09:56:01.500 +05:30 [ERR] Connection failed. Retrying in 5 seconds... Attempt 42
2025-04-02 09:56:01.501 +05:30 [ERR] Exception: 42P01: relation "inserted_invoices" does not exist
2025-04-02 09:56:07.126 +05:30 [ERR] Connection failed. Retrying in 5 seconds... Attempt 43
2025-04-02 09:56:07.128 +05:30 [ERR] Exception: 42P01: relation "inserted_invoices" does not exist
2025-04-02 09:56:12.521 +05:30 [ERR] Connection failed. Retrying in 5 seconds... Attempt 44
2025-04-02 09:56:12.521 +05:30 [ERR] Exception: 42P01: relation "inserted_invoices" does not exist
2025-04-02 09:56:17.928 +05:30 [ERR] Connection failed. Retrying in 5 seconds... Attempt 45
2025-04-02 09:56:17.929 +05:30 [ERR] Exception: 42P01: relation "inserted_invoices" does not exist
2025-04-02 09:56:23.315 +05:30 [ERR] Connection failed. Retrying in 5 seconds... Attempt 46
2025-04-02 09:56:23.317 +05:30 [ERR] Exception: 42P01: relation "inserted_invoices" does not exist
2025-04-02 09:56:28.736 +05:30 [ERR] Connection failed. Retrying in 5 seconds... Attempt 47
2025-04-02 09:56:28.738 +05:30 [ERR] Exception: 42P01: relation "inserted_invoices" does not exist
2025-04-02 09:56:34.145 +05:30 [ERR] Connection failed. Retrying in 5 seconds... Attempt 48
2025-04-02 09:56:34.146 +05:30 [ERR] Exception: 42P01: relation "inserted_invoices" does not exist
2025-04-02 09:56:39.535 +05:30 [ERR] Connection failed. Retrying in 5 seconds... Attempt 49
2025-04-02 09:56:39.536 +05:30 [ERR] Exception: 42P01: relation "inserted_invoices" does not exist
2025-04-02 09:56:44.916 +05:30 [ERR] Connection failed. Retrying in 5 seconds... Attempt 50
2025-04-02 09:56:44.916 +05:30 [ERR] Exception: 42P01: relation "inserted_invoices" does not exist
2025-04-02 09:56:50.290 +05:30 [ERR] Connection failed. Retrying in 5 seconds... Attempt 51
2025-04-02 09:56:50.291 +05:30 [ERR] Exception: 42P01: relation "inserted_invoices" does not exist
2025-04-02 09:56:55.690 +05:30 [ERR] Connection failed. Retrying in 5 seconds... Attempt 52
2025-04-02 09:56:55.691 +05:30 [ERR] Exception: 42P01: relation "inserted_invoices" does not exist
2025-04-02 09:57:01.060 +05:30 [ERR] Connection failed. Retrying in 5 seconds... Attempt 53
2025-04-02 09:57:01.060 +05:30 [ERR] Exception: 42P01: relation "inserted_invoices" does not exist
2025-04-02 09:57:06.456 +05:30 [ERR] Connection failed. Retrying in 5 seconds... Attempt 54
2025-04-02 09:57:06.457 +05:30 [ERR] Exception: 42P01: relation "inserted_invoices" does not exist
2025-04-02 09:57:11.812 +05:30 [ERR] Connection failed. Retrying in 5 seconds... Attempt 55
2025-04-02 09:57:11.812 +05:30 [ERR] Exception: 42P01: relation "inserted_invoices" does not exist
2025-04-02 09:57:17.197 +05:30 [ERR] Connection failed. Retrying in 5 seconds... Attempt 56
2025-04-02 09:57:17.198 +05:30 [ERR] Exception: 42P01: relation "inserted_invoices" does not exist
2025-04-02 09:57:23.099 +05:30 [ERR] Connection failed. Retrying in 5 seconds... Attempt 57
2025-04-02 09:57:23.101 +05:30 [ERR] Exception: 42P01: relation "inserted_invoices" does not exist
2025-04-02 09:57:28.440 +05:30 [ERR] Connection failed. Retrying in 5 seconds... Attempt 58
2025-04-02 09:57:28.441 +05:30 [ERR] Exception: 42P01: relation "inserted_invoices" does not exist
2025-04-02 09:57:33.728 +05:30 [INF] Process file path :- C:\BackOffice\ETL_Replacement\PIL\INTEL\Input\Source\Temp_Process\INTEL_01_v1_20231208_TELINV04_WIP.csv
2025-04-02 09:57:33.728 +05:30 [INF] Process file name :- INTEL_01_v1_20231208_TELINV04_WIP.csv
2025-04-02 09:57:33.729 +05:30 [INF] Total time taken in process: 316.701182 seconds
2025-04-02 09:57:33.731 +05:30 [INF] Total time taken in moving: 0.002259 seconds
2025-04-02 09:57:40.837 +05:30 [INF] Application starting up
2025-04-02 09:57:41.681 +05:30 [INF] Main thread ID: 1
2025-04-02 09:57:41.681 +05:30 [INF] ARGS:- C:\BackOffice\ETL_Replacement\PIL\INTEL\Input\Source\Temp_Process\INTEL_01_v1_20231208_TELINV04_WIP.csv
2025-04-02 09:57:41.684 +05:30 [INF] Processing files for domain: PIL, invoiceType: INTEL
2025-04-02 09:57:43.210 +05:30 [INF] Exception in ProcessTaskAsync : Could not find file 'C:\BackOffice\ETL_Replacement\PIL\INTEL\Input\Source\Temp_Process\INTEL_01_v1_20231208_TELINV04_WIP.csv'.
2025-04-02 09:57:43.283 +05:30 [INF] Exception in ProcessFileAsync : Could not find file 'C:\BackOffice\ETL_Replacement\PIL\INTEL\Input\Source\Temp_Process\INTEL_01_v1_20231208_TELINV04_WIP.csv'.
2025-04-02 09:57:43.348 +05:30 [INF] Error processing files for domain PIL: Could not find file 'C:\BackOffice\ETL_Replacement\PIL\INTEL\Input\Source\Temp_Process\INTEL_01_v1_20231208_TELINV04_WIP.csv'.
2025-04-02 09:57:43.349 +05:30 [INF] The following files were not found and skipped:
2025-04-02 09:57:43.349 +05:30 [INF] C:\BackOffice\ETL_Replacement\PIL\INTEL\Input\Source\Temp_Process\INTEL_01_v1_20231208_TELINV04_WIP.csv
2025-04-02 09:58:02.683 +05:30 [INF] Application starting up
2025-04-02 09:58:03.412 +05:30 [INF] Main thread ID: 1
2025-04-02 09:58:03.412 +05:30 [INF] ARGS:- C:\BackOffice\ETL_Replacement\PIL\INTEL\Input\Source\Temp_Process\INTEL_01_v1_20231208_TELINV04_WIP.csv
2025-04-02 09:58:03.415 +05:30 [INF] Processing files for domain: PIL, invoiceType: INTEL
2025-04-02 09:58:08.026 +05:30 [INF] Process file path :- C:\BackOffice\ETL_Replacement\PIL\INTEL\Input\Source\Temp_Process\INTEL_01_v1_20231208_TELINV04_WIP.csv
2025-04-02 09:58:08.026 +05:30 [INF] Process file name :- INTEL_01_v1_20231208_TELINV04_WIP.csv
2025-04-02 09:58:08.026 +05:30 [INF] Total time taken in process: 2.6944823 seconds
2025-04-02 09:58:08.029 +05:30 [INF] Total time taken in moving: 0.0024475 seconds
2025-04-02 10:01:09.555 +05:30 [INF] Application starting up
2025-04-02 10:01:10.321 +05:30 [INF] Main thread ID: 1
2025-04-02 10:01:10.322 +05:30 [INF] ARGS:- C:\BackOffice\ETL_Replacement\PIL\INTEL\Input\Source\Temp_Process\INTEL_01_v1_20231208_TELINV04_WIP.csv
2025-04-02 10:01:10.325 +05:30 [INF] Processing files for domain: PIL, invoiceType: INTEL
2025-04-02 10:01:13.347 +05:30 [ERR] Connection failed. Retrying in 5 seconds... Attempt 1
2025-04-02 10:01:13.347 +05:30 [ERR] Exception: 42703: column "accountnumber" does not exist
2025-04-02 10:05:19.694 +05:30 [INF] Application starting up
2025-04-02 10:05:20.612 +05:30 [INF] Main thread ID: 1
2025-04-02 10:05:20.612 +05:30 [INF] ARGS:- C:\BackOffice\ETL_Replacement\PIL\INTEL\Input\Source\Temp_Process\INTEL_01_v1_20231208_TELINV04_WIP.csv
2025-04-02 10:05:20.615 +05:30 [INF] Processing files for domain: PIL, invoiceType: INTEL
2025-04-02 10:05:23.771 +05:30 [ERR] Connection failed. Retrying in 5 seconds... Attempt 1
2025-04-02 10:05:23.771 +05:30 [ERR] Exception: 42703: column "accountnumber" does not exist
2025-04-02 10:06:53.697 +05:30 [INF] Application starting up
2025-04-02 10:06:54.413 +05:30 [INF] Main thread ID: 1
2025-04-02 10:06:54.414 +05:30 [INF] ARGS:- C:\BackOffice\ETL_Replacement\PIL\INTEL\Input\Source\Temp_Process\INTEL_01_v1_20231208_TELINV04_WIP.csv
2025-04-02 10:06:54.416 +05:30 [INF] Processing files for domain: PIL, invoiceType: INTEL
2025-04-02 10:06:57.515 +05:30 [ERR] Connection failed. Retrying in 5 seconds... Attempt 1
2025-04-02 10:06:57.515 +05:30 [ERR] Exception: 42703: column "accountnumber" does not exist
2025-04-02 10:07:02.864 +05:30 [ERR] Connection failed. Retrying in 5 seconds... Attempt 2
2025-04-02 10:07:02.865 +05:30 [ERR] Exception: 42703: column "accountnumber" does not exist
2025-04-02 10:07:08.227 +05:30 [ERR] Connection failed. Retrying in 5 seconds... Attempt 3
2025-04-02 10:07:08.228 +05:30 [ERR] Exception: 42703: column "accountnumber" does not exist
2025-04-02 10:07:13.537 +05:30 [ERR] Connection failed. Retrying in 5 seconds... Attempt 4
2025-04-02 10:07:13.537 +05:30 [ERR] Exception: 42703: column "accountnumber" does not exist
2025-04-02 10:07:19.161 +05:30 [ERR] Connection failed. Retrying in 5 seconds... Attempt 5
2025-04-02 10:07:19.162 +05:30 [ERR] Exception: 42703: column "accountnumber" does not exist
2025-04-02 10:07:24.392 +05:30 [ERR] Connection failed. Retrying in 5 seconds... Attempt 6
2025-04-02 10:07:24.393 +05:30 [ERR] Exception: 42703: column "accountnumber" does not exist
2025-04-02 10:07:29.797 +05:30 [ERR] Connection failed. Retrying in 5 seconds... Attempt 7
2025-04-02 10:07:29.798 +05:30 [ERR] Exception: 42703: column "accountnumber" does not exist
2025-04-02 10:07:35.048 +05:30 [ERR] Connection failed. Retrying in 5 seconds... Attempt 8
2025-04-02 10:07:35.049 +05:30 [ERR] Exception: 42703: column "accountnumber" does not exist
2025-04-02 10:07:40.415 +05:30 [ERR] Connection failed. Retrying in 5 seconds... Attempt 9
2025-04-02 10:07:40.416 +05:30 [ERR] Exception: 42703: column "accountnumber" does not exist
2025-04-02 10:07:45.723 +05:30 [ERR] Connection failed. Retrying in 5 seconds... Attempt 10
2025-04-02 10:07:45.724 +05:30 [ERR] Exception: 42703: column "accountnumber" does not exist
2025-04-02 10:07:51.009 +05:30 [ERR] Connection failed. Retrying in 5 seconds... Attempt 11
2025-04-02 10:07:51.009 +05:30 [ERR] Exception: 42703: column "accountnumber" does not exist
2025-04-02 10:07:56.292 +05:30 [ERR] Connection failed. Retrying in 5 seconds... Attempt 12
2025-04-02 10:07:56.292 +05:30 [ERR] Exception: 42703: column "accountnumber" does not exist
2025-04-02 10:08:01.580 +05:30 [ERR] Connection failed. Retrying in 5 seconds... Attempt 13
2025-04-02 10:08:01.580 +05:30 [ERR] Exception: 42703: column "accountnumber" does not exist
2025-04-02 10:08:06.929 +05:30 [ERR] Connection failed. Retrying in 5 seconds... Attempt 14
2025-04-02 10:08:06.930 +05:30 [ERR] Exception: 42703: column "accountnumber" does not exist
2025-04-02 10:08:12.243 +05:30 [ERR] Connection failed. Retrying in 5 seconds... Attempt 15
2025-04-02 10:08:12.244 +05:30 [ERR] Exception: 42703: column "accountnumber" does not exist
2025-04-02 10:08:17.590 +05:30 [ERR] Connection failed. Retrying in 5 seconds... Attempt 16
2025-04-02 10:08:17.591 +05:30 [ERR] Exception: 42703: column "accountnumber" does not exist
2025-04-02 10:08:22.905 +05:30 [ERR] Connection failed. Retrying in 5 seconds... Attempt 17
2025-04-02 10:08:22.905 +05:30 [ERR] Exception: 42703: column "accountnumber" does not exist
2025-04-02 10:08:28.255 +05:30 [ERR] Connection failed. Retrying in 5 seconds... Attempt 18
2025-04-02 10:08:28.256 +05:30 [ERR] Exception: 42703: column "accountnumber" does not exist
2025-04-02 10:08:33.820 +05:30 [ERR] Connection failed. Retrying in 5 seconds... Attempt 19
2025-04-02 10:08:33.821 +05:30 [ERR] Exception: 42703: column "accountnumber" does not exist
2025-04-02 10:08:39.278 +05:30 [ERR] Connection failed. Retrying in 5 seconds... Attempt 20
2025-04-02 10:08:39.279 +05:30 [ERR] Exception: 42703: column "accountnumber" does not exist
2025-04-02 10:08:44.637 +05:30 [ERR] Connection failed. Retrying in 5 seconds... Attempt 21
2025-04-02 10:08:44.637 +05:30 [ERR] Exception: 42703: column "accountnumber" does not exist
2025-04-02 10:08:50.031 +05:30 [ERR] Connection failed. Retrying in 5 seconds... Attempt 22
2025-04-02 10:08:50.032 +05:30 [ERR] Exception: 42703: column "accountnumber" does not exist
2025-04-02 10:08:55.438 +05:30 [INF] Process file path :- C:\BackOffice\ETL_Replacement\PIL\INTEL\Input\Source\Temp_Process\INTEL_01_v1_20231208_TELINV04_WIP.csv
2025-04-02 10:08:55.438 +05:30 [INF] Process file name :- INTEL_01_v1_20231208_TELINV04_WIP.csv
2025-04-02 10:08:55.439 +05:30 [INF] Total time taken in process: 119.7120257 seconds
2025-04-02 10:08:55.441 +05:30 [INF] Total time taken in moving: 0.0021952 seconds
2025-04-02 10:09:19.671 +05:30 [INF] Application starting up
2025-04-02 10:09:20.740 +05:30 [INF] Main thread ID: 1
2025-04-02 10:09:20.740 +05:30 [INF] ARGS:- C:\BackOffice\ETL_Replacement\PIL\INTEL\Input\Source\Temp_Process\INTEL_01_v1_20231208_TELINV04_WIP.csv
2025-04-02 10:09:20.744 +05:30 [INF] Processing files for domain: PIL, invoiceType: INTEL
2025-04-02 10:09:22.404 +05:30 [INF] Exception in ProcessTaskAsync : Could not find file 'C:\BackOffice\ETL_Replacement\PIL\INTEL\Input\Source\Temp_Process\INTEL_01_v1_20231208_TELINV04_WIP.csv'.
2025-04-02 10:09:22.473 +05:30 [INF] Exception in ProcessFileAsync : Could not find file 'C:\BackOffice\ETL_Replacement\PIL\INTEL\Input\Source\Temp_Process\INTEL_01_v1_20231208_TELINV04_WIP.csv'.
2025-04-02 10:09:22.535 +05:30 [INF] Error processing files for domain PIL: Could not find file 'C:\BackOffice\ETL_Replacement\PIL\INTEL\Input\Source\Temp_Process\INTEL_01_v1_20231208_TELINV04_WIP.csv'.
2025-04-02 10:09:22.535 +05:30 [INF] The following files were not found and skipped:
2025-04-02 10:09:22.535 +05:30 [INF] C:\BackOffice\ETL_Replacement\PIL\INTEL\Input\Source\Temp_Process\INTEL_01_v1_20231208_TELINV04_WIP.csv
2025-04-02 10:09:40.775 +05:30 [INF] Application starting up
2025-04-02 10:09:41.576 +05:30 [INF] Main thread ID: 1
2025-04-02 10:09:41.576 +05:30 [INF] ARGS:- C:\BackOffice\ETL_Replacement\PIL\INTEL\Input\Source\Temp_Process\INTEL_01_v1_20231208_TELINV04_WIP.csv
2025-04-02 10:09:41.579 +05:30 [INF] Processing files for domain: PIL, invoiceType: INTEL
2025-04-02 10:09:44.352 +05:30 [INF] Process file path :- C:\BackOffice\ETL_Replacement\PIL\INTEL\Input\Source\Temp_Process\INTEL_01_v1_20231208_TELINV04_WIP.csv
2025-04-02 10:09:44.352 +05:30 [INF] Process file name :- INTEL_01_v1_20231208_TELINV04_WIP.csv
2025-04-02 10:09:44.353 +05:30 [INF] Total time taken in process: 1.3756663 seconds
2025-04-02 10:09:44.356 +05:30 [INF] Total time taken in moving: 0.0026207 seconds
2025-04-02 10:13:08.596 +05:30 [INF] Application starting up
2025-04-02 10:13:09.813 +05:30 [INF] Main thread ID: 1
2025-04-02 10:13:09.813 +05:30 [INF] ARGS:- C:\BackOffice\ETL_Replacement\PIL\INTEL\Input\Source\Temp_Process\INTEL_01_v1_20231208_TELINV04_WIP.csv
2025-04-02 10:13:09.815 +05:30 [INF] Processing files for domain: PIL, invoiceType: INTEL
2025-04-02 10:13:13.019 +05:30 [INF] Process file path :- C:\BackOffice\ETL_Replacement\PIL\INTEL\Input\Source\Temp_Process\INTEL_01_v1_20231208_TELINV04_WIP.csv
2025-04-02 10:13:13.019 +05:30 [INF] Process file name :- INTEL_01_v1_20231208_TELINV04_WIP.csv
2025-04-02 10:13:13.020 +05:30 [INF] Total time taken in process: 1.8602615 seconds
2025-04-02 10:13:13.023 +05:30 [INF] Total time taken in moving: 0.0027221 seconds
2025-04-02 10:32:56.093 +05:30 [INF] Application starting up
2025-04-02 10:32:56.917 +05:30 [INF] Main thread ID: 1
2025-04-02 10:32:56.917 +05:30 [INF] ARGS:- C:\BackOffice\ETL_Replacement\PIL\INTEL\Input\Source\Temp_Process\INTEL_01_v1_20231208_TELINV04_WIP.csv
2025-04-02 10:32:56.919 +05:30 [INF] Processing files for domain: PIL, invoiceType: INTEL
2025-04-02 10:33:00.025 +05:30 [ERR] Connection failed. Retrying in 5 seconds... Attempt 1
2025-04-02 10:33:00.025 +05:30 [ERR] Exception: 42P01: relation "inserted_invoices" does not exist
2025-04-02 10:34:27.487 +05:30 [INF] Application starting up
2025-04-02 10:34:28.278 +05:30 [INF] Main thread ID: 1
2025-04-02 10:34:28.278 +05:30 [INF] ARGS:- C:\BackOffice\ETL_Replacement\PIL\INTEL\Input\Source\Temp_Process\INTEL_01_v1_20231208_TELINV04_WIP.csv
2025-04-02 10:34:28.280 +05:30 [INF] Processing files for domain: PIL, invoiceType: INTEL
2025-04-02 10:34:31.135 +05:30 [ERR] Connection failed. Retrying in 5 seconds... Attempt 1
2025-04-02 10:34:31.135 +05:30 [ERR] Exception: 42703: column ii.cbcinvoicetypecode does not exist
2025-04-02 10:35:20.780 +05:30 [INF] Application starting up
2025-04-02 10:35:21.733 +05:30 [INF] Main thread ID: 1
2025-04-02 10:35:21.734 +05:30 [INF] ARGS:- C:\BackOffice\ETL_Replacement\PIL\INTEL\Input\Source\Temp_Process\INTEL_01_v1_20231208_TELINV04_WIP.csv
2025-04-02 10:35:21.742 +05:30 [INF] Processing files for domain: PIL, invoiceType: INTEL
2025-04-02 10:35:24.784 +05:30 [ERR] Connection failed. Retrying in 5 seconds... Attempt 1
2025-04-02 10:35:24.784 +05:30 [ERR] Exception: 42703: column ii.cbcinvoicetypecode does not exist
2025-04-02 10:35:30.067 +05:30 [ERR] Connection failed. Retrying in 5 seconds... Attempt 2
2025-04-02 10:35:30.067 +05:30 [ERR] Exception: 42703: column ii.cbcinvoicetypecode does not exist
2025-04-02 10:36:16.312 +05:30 [INF] Application starting up
2025-04-02 10:36:17.216 +05:30 [INF] Main thread ID: 1
2025-04-02 10:36:17.216 +05:30 [INF] ARGS:- C:\BackOffice\ETL_Replacement\PIL\INTEL\Input\Source\Temp_Process\INTEL_01_v1_20231208_TELINV04_WIP.csv
2025-04-02 10:36:17.224 +05:30 [INF] Processing files for domain: PIL, invoiceType: INTEL
2025-04-02 10:36:20.294 +05:30 [ERR] Connection failed. Retrying in 5 seconds... Attempt 1
2025-04-02 10:36:20.294 +05:30 [ERR] Exception: 42P01: missing FROM-clause entry for table "ii"
2025-04-02 10:36:25.699 +05:30 [ERR] Connection failed. Retrying in 5 seconds... Attempt 2
2025-04-02 10:36:25.699 +05:30 [ERR] Exception: 42P01: missing FROM-clause entry for table "ii"
2025-04-02 10:37:36.864 +05:30 [INF] Application starting up
2025-04-02 10:37:37.528 +05:30 [INF] Main thread ID: 1
2025-04-02 10:37:37.528 +05:30 [INF] ARGS:- C:\BackOffice\ETL_Replacement\PIL\INTEL\Input\Source\Temp_Process\INTEL_01_v1_20231208_TELINV04_WIP.csv
2025-04-02 10:37:37.531 +05:30 [INF] Processing files for domain: PIL, invoiceType: INTEL
2025-04-02 10:37:40.308 +05:30 [ERR] Connection failed. Retrying in 5 seconds... Attempt 1
2025-04-02 10:37:40.308 +05:30 [ERR] Exception: 0A000: WITH query "insert_pdf_inv" does not have a RETURNING clause
2025-04-02 10:38:28.746 +05:30 [INF] Application starting up
2025-04-02 10:38:29.730 +05:30 [INF] Main thread ID: 1
2025-04-02 10:38:29.730 +05:30 [INF] ARGS:- C:\BackOffice\ETL_Replacement\PIL\INTEL\Input\Source\Temp_Process\INTEL_01_v1_20231208_TELINV04_WIP.csv
2025-04-02 10:38:29.732 +05:30 [INF] Processing files for domain: PIL, invoiceType: INTEL
2025-04-02 10:38:33.207 +05:30 [ERR] Connection failed. Retrying in 5 seconds... Attempt 1
2025-04-02 10:38:33.208 +05:30 [ERR] Exception: 42702: column reference "invoicenumber" is ambiguous

DETAIL: Detail redacted as it may contain sensitive data. Specify 'Include Error Detail' in the connection string to include this information.
2025-04-02 10:39:14.337 +05:30 [INF] Application starting up
2025-04-02 10:39:15.072 +05:30 [INF] Main thread ID: 1
2025-04-02 10:39:15.073 +05:30 [INF] ARGS:- C:\BackOffice\ETL_Replacement\PIL\INTEL\Input\Source\Temp_Process\INTEL_01_v1_20231208_TELINV04_WIP.csv
2025-04-02 10:39:15.075 +05:30 [INF] Processing files for domain: PIL, invoiceType: INTEL
2025-04-02 10:39:18.251 +05:30 [ERR] Connection failed. Retrying in 5 seconds... Attempt 1
2025-04-02 10:39:18.251 +05:30 [ERR] Exception: 42702: column reference "invoicenumber" is ambiguous

DETAIL: Detail redacted as it may contain sensitive data. Specify 'Include Error Detail' in the connection string to include this information.
2025-04-02 10:41:26.838 +05:30 [INF] Application starting up
2025-04-02 10:41:27.523 +05:30 [INF] Main thread ID: 1
2025-04-02 10:41:27.523 +05:30 [INF] ARGS:- C:\BackOffice\ETL_Replacement\PIL\INTEL\Input\Source\Temp_Process\INTEL_01_v1_20231208_TELINV04_WIP.csv
2025-04-02 10:41:27.525 +05:30 [INF] Processing files for domain: PIL, invoiceType: INTEL
2025-04-02 10:41:30.622 +05:30 [ERR] Connection failed. Retrying in 5 seconds... Attempt 1
2025-04-02 10:41:30.622 +05:30 [ERR] Exception: 42703: column ii.cbcinvoicetypecode does not exist
2025-04-02 10:42:34.223 +05:30 [INF] Application starting up
2025-04-02 10:42:35.248 +05:30 [INF] Main thread ID: 1
2025-04-02 10:42:35.248 +05:30 [INF] ARGS:- C:\BackOffice\ETL_Replacement\PIL\INTEL\Input\Source\Temp_Process\INTEL_01_v1_20231208_TELINV04_WIP.csv
2025-04-02 10:42:35.250 +05:30 [INF] Processing files for domain: PIL, invoiceType: INTEL
2025-04-02 10:42:38.198 +05:30 [ERR] Connection failed. Retrying in 5 seconds... Attempt 1
2025-04-02 10:42:38.198 +05:30 [ERR] Exception: 42703: column i2.cbcinvoicetypecode does not exist
2025-04-02 10:45:14.990 +05:30 [INF] Application starting up
2025-04-02 10:45:15.669 +05:30 [INF] Main thread ID: 1
2025-04-02 10:45:15.669 +05:30 [INF] ARGS:- C:\BackOffice\ETL_Replacement\PIL\INTEL\Input\Source\Temp_Process\INTEL_01_v1_20231208_TELINV04_WIP.csv
2025-04-02 10:45:15.672 +05:30 [INF] Processing files for domain: PIL, invoiceType: INTEL
2025-04-02 10:45:19.650 +05:30 [ERR] Connection failed. Retrying in 5 seconds... Attempt 1
2025-04-02 10:45:19.650 +05:30 [ERR] Exception: 42804: column "register" is of type date but expression is of type text
2025-04-02 10:45:25.100 +05:30 [ERR] Connection failed. Retrying in 5 seconds... Attempt 2
2025-04-02 10:45:25.100 +05:30 [ERR] Exception: 42804: column "register" is of type date but expression is of type text
2025-04-02 10:45:58.105 +05:30 [INF] Application starting up
2025-04-02 10:45:58.792 +05:30 [INF] Main thread ID: 1
2025-04-02 10:45:58.792 +05:30 [INF] ARGS:- C:\BackOffice\ETL_Replacement\PIL\INTEL\Input\Source\Temp_Process\INTEL_01_v1_20231208_TELINV04_WIP.csv
2025-04-02 10:45:58.796 +05:30 [INF] Processing files for domain: PIL, invoiceType: INTEL
2025-04-02 10:46:01.583 +05:30 [INF] Process file path :- C:\BackOffice\ETL_Replacement\PIL\INTEL\Input\Source\Temp_Process\INTEL_01_v1_20231208_TELINV04_WIP.csv
2025-04-02 10:46:01.583 +05:30 [INF] Process file name :- INTEL_01_v1_20231208_TELINV04_WIP.csv
2025-04-02 10:46:01.583 +05:30 [INF] Total time taken in process: 1.4233931 seconds
2025-04-02 10:46:01.586 +05:30 [INF] Total time taken in moving: 0.0025974 seconds
2025-04-02 10:53:48.897 +05:30 [INF] Application starting up
2025-04-02 10:53:49.591 +05:30 [INF] Main thread ID: 1
2025-04-02 10:53:49.592 +05:30 [INF] ARGS:- C:\BackOffice\ETL_Replacement\PIL\INTEL\Input\Source\Temp_Process\INTEL_01_v1_20231208_TELINV04_WIP.csv
2025-04-02 10:53:49.594 +05:30 [INF] Processing files for domain: PIL, invoiceType: INTEL
2025-04-02 10:53:53.144 +05:30 [ERR] Connection failed. Retrying in 5 seconds... Attempt 1
2025-04-02 10:53:53.144 +05:30 [ERR] Exception: 0A000: WITH query "inserted_line_items" does not have a RETURNING clause
2025-04-02 10:55:01.275 +05:30 [INF] Application starting up
2025-04-02 10:55:02.019 +05:30 [INF] Main thread ID: 1
2025-04-02 10:55:02.019 +05:30 [INF] ARGS:- C:\BackOffice\ETL_Replacement\PIL\INTEL\Input\Source\Temp_Process\INTEL_01_v1_20231208_TELINV04_WIP.csv
2025-04-02 10:55:02.021 +05:30 [INF] Processing files for domain: PIL, invoiceType: INTEL
2025-04-02 10:55:04.877 +05:30 [ERR] Connection failed. Retrying in 5 seconds... Attempt 1
2025-04-02 10:55:04.878 +05:30 [ERR] Exception: 42703: column i3.invoiceid does not exist
2025-04-02 10:55:29.768 +05:30 [INF] Application starting up
2025-04-02 10:55:30.742 +05:30 [INF] Main thread ID: 1
2025-04-02 10:55:30.742 +05:30 [INF] ARGS:- C:\BackOffice\ETL_Replacement\PIL\INTEL\Input\Source\Temp_Process\INTEL_01_v1_20231208_TELINV04_WIP.csv
2025-04-02 10:55:30.750 +05:30 [INF] Processing files for domain: PIL, invoiceType: INTEL
2025-04-02 10:55:34.499 +05:30 [ERR] Connection failed. Retrying in 5 seconds... Attempt 1
2025-04-02 10:55:34.499 +05:30 [ERR] Exception: 42702: column reference "invoiceid" is ambiguous

DETAIL: Detail redacted as it may contain sensitive data. Specify 'Include Error Detail' in the connection string to include this information.
2025-04-02 10:56:55.666 +05:30 [INF] Application starting up
2025-04-02 10:56:55.956 +05:30 [INF] Main thread ID: 1
2025-04-02 10:56:55.956 +05:30 [INF] ARGS:- C:\BackOffice\ETL_Replacement\PIL\INTEL\Input\Source\Temp_Process\INTEL_01_v1_20231208_TELINV04_WIP.csv
2025-04-02 10:56:55.959 +05:30 [INF] Processing files for domain: PIL, invoiceType: INTEL
2025-04-02 10:56:58.223 +05:30 [ERR] Connection failed. Retrying in 5 seconds... Attempt 1
2025-04-02 10:56:58.223 +05:30 [ERR] Exception: 42702: column reference "invoiceid" is ambiguous

DETAIL: Detail redacted as it may contain sensitive data. Specify 'Include Error Detail' in the connection string to include this information.
